src/Scripting/CSharpTest/CommandLineRunnerTests.cs

```diff
@@ -132,14 +132,19 @@ public void Await()
 > ", runner.Console.Out.ToString());
 
             AssertEx.AssertEqualToleratingWhitespaceDifferences(
-                @"(1,19): warning CS1998: This async method lacks 'await' operators and will run synchronously. Consider using the 'await' operator to await non-blocking API calls, or 'await Task.Run(...)' to do CPU-bound work on a background thread.",
+                $@"(1,19): warning CS1998: { CSharpResources.WRN_AsyncLacksAwaits }",
                 runner.Console.Error.ToString());
         }
 
-        [Fact(Skip="https://github.com/dotnet/roslyn/issues/17043")]
+        [Fact(Skip = "https://github.com/dotnet/roslyn/issues/17043")]
         [WorkItem(7133, "http://github.com/dotnet/roslyn/issues/7133")]
-        public void TestDisplayResultsWithCurrentUICulture()
+        public void TestDisplayResultsWithCurrentUICulture1()
         {
+            // logoOutput needs to be retrieved before the runner is started, because the runner changes the culture to de-DE. 
+            var logoOutput = $@"{ string.Format(CSharpScriptingResources.LogoLine1, s_compilerVersion) }
+{ CSharpScriptingResources.LogoLine2}
+
+{ ScriptingResources.HelpPrompt}";
```

Comment:
Could you extract to a static field (similar to s_compilerVersion) and then use the field in all test cases?